.gitmodules

```diff
@@ -1,8 +1,12 @@
-[submodule "third-party/googletest"]
-	path = third-party/googletest
-	url = https://github.com/google/googletest.git
-	branch = v1.14.x
+[submodule "third-party/boost"]
+	path = third-party/boost
+	url = https://github.com/boostorg/boost.git
+	branch = master
 [submodule "third-party/json"]
 	path = third-party/json
 	url = https://github.com/nlohmann/json.git
 	branch = master
+[submodule "third-party/googletest"]
+	path = third-party/googletest
+	url = https://github.com/google/googletest.git
+	branch = v1.14.x
```

Comment:
Can this go back above `third-party/json`?

Reply:
Ugh, RIP my alphabet skills...